### ISSUE TITLE
[Feature] Support configuration with environment variables

### ISSUE THREAD

**Issue by forzagreen** [utkarshcmu/wizzy#146]
It would be nice if wizzy can load json files containing environment variables. This is useful for conf/wizzy.json and datasources.
The `conf/wizzy.json` file would look like:

```json
{
  "config": {
    "grafana": {
      "url": "https://mygrafana.example.com",
      "username": "${ADMIN_USERNAME}",
      "password": "${ADMIN_PASSWORD}"
    }
  }
}
```

Example of a datasource `datasources/my-influxdb.json`:

```json
{
  "orgId": 1,
  "name": "my-influxdb",
  "type": "influxdb",
  "url": "http://localhost:8086",
  "user": "${DB_USER}",
  "password": "${DB_PASSWORD}",
  "database": "${DB_NAME}"
}
```

It can achieved e.g. with [Mustache.js](https://github.com/janl/mustache.js).

FYI, Grafana support using environment variables in its config file: cf. https://grafana.com/docs/installation/configuration/#using-environment-variables

**Comment by Sytten** [comment]
I also agree with this idea, I have some dashboards that could use variables before export.

**Comment by johnsudaar** [comment]
I thought of that too. It would be really nice. Actually I'm asking if integrating an entire templating engine wouldn't be nice. Having thinks like conditions loops and support for environment variables would be really nice.

**Comment by johnsudaar** [comment]
I think that i'll try to PoC adding a templating engine to the current js files.

**Comment by Sytten** [comment]
Postponed to 0.80
